src/screens/Ticketing/Ticket/ValidityLine.tsx

```diff
@@ -57,4 +57,104 @@ const ValidityLine: React.FC<{
   }
 };
 
+/**
+ * Starts an animation for the vertical line offset which loops continuously.
+ * The offset is animated from 0 to 1, which must be interpolated to the actual
+ * pixel offset value in the vertical line component.
+ * @returns the current animated offset value
+ */
+const useAnimatedVerticalLineOffset = () => {
+  const animatedOffset = useRef(new Animated.Value(0)).current;
+  useEffect(() => {
+    Animated.loop(
```

Comment:
Should return in useEffect from Animated.loop.start(), since the return value from start() is a function to stop the animation. Potentially it could loop forever even if the screen is unmounted/ticket is no longer valid.
```suggestion
    return Animated.loop(
```

Reply:
Fixed